### ISSUE TITLE
Broken links on runtime spec

### ISSUE THREAD

**Issue by Jamstah** [opencontainers/specs.opencontainers.org#3]
https://specs.opencontainers.org/runtime-spec/

Links to anything under [Runtime and Lifecycle](https://specs.opencontainers.org/runtime-spec/runtime.md) or [Configuration](https://specs.opencontainers.org/runtime-spec/config.md) are broken.

**Comment by vsoch** [comment]
It looks like the parser is likely choking on something here: https://raw.githubusercontent.com/opencontainers/runtime-spec/v1.0.2/spec.md. We need to add more verbosity to printing what is happening on a local deployment to debug.

**Comment by vsoch** [comment]
The issue seems to be this line:

```
html = html.replace(link, {% if page.parent %}"{{ site.baseurl }}/{{ page.parent }}/" + {% endif %} basename(link).replace(".md") + "/?v=" + window.version);
```
Given config.md for the link text, that would replace the first occurrence in the html. The problem is that there are many redundant occurrences of the string, so likely it's replaced elsewhere and missed for the actual link.

**Comment by vsoch** [comment]
We can see more closely what happens here, before and after parsing "config.md" via that means.

Here is before - note `<a href="config/?v=v1.0.2">config.md</a>`

```html
<li><code>linux</code>: <a href="runtime/?v=v1.0.2">runtime/?v=v1.0.2</a>, <a href="config/?v=v1.0.2">config/?v=v1.0.2</a>, <a href="config-linux/?v=v1.0.2">config-linux.md</a>, and <a href="runtime-linux/?v=v1.0.2">runtime-linux.md</a>.</li>
<li><code>solaris</code>: <a href="runtime/?v=v1.0.2">runtime/?v=v1.0.2</a>, <a href="config/?v=v1.0.2">config.md</a>, and <a href="config-solaris/?v=v1.0.2">config-solaris.md</a>.</li>
<li><code>windows</code>: <a href="runtime.md">runtime.md</a>, <a href="config.md">config.md</a>, and <a href="config-windows/?v=v1.0.2">config-windows.md</a>.</li>
<li><code>vm</code>: <a href="runtime.md">runtime.md</a>, <a href="config.md">config.md</a>, and <a href="config-vm.md">config-vm.md</a>.</li>
```
And here is after - note `<a href="config/?v=v1.0.2">config/?v=v1.0.2</a>` where the replacement happens

```html
<li><code>linux</code>: <a href="runtime/?v=v1.0.2">runtime/?v=v1.0.2</a>, <a href="config/?v=v1.0.2">config/?v=v1.0.2</a>, <a href="config-linux/?v=v1.0.2">config-linux.md</a>, and <a href="runtime-linux/?v=v1.0.2">runtime-linux.md</a>.</li>
<li><code>solaris</code>: <a href="runtime/?v=v1.0.2">runtime/?v=v1.0.2</a>, <a href="config/?v=v1.0.2">config/?v=v1.0.2</a>, and <a href="config-solaris/?v=v1.0.2">config-solaris.md</a>.</li>
<li><code>windows</code>: <a href="runtime.md">runtime.md</a>, <a href="config.md">config.md</a>, and <a href="config-windows/?v=v1.0.2">config-windows.md</a>.</li>
<li><code>vm</code>: <a href="runtime.md">runtime.md</a>, <a href="config.md">config.md</a>, and <a href="config-vm.md">config-vm.md</a>.</li>
```
So the replacement does happen, but since the link text is the same as the link (the markdown name) it's replacing that instead. Suggestions for fixes welcome - I'm certainly not a web developer / frontend person.

**Comment by amye** [comment]
@nate-double-u - any idea of how we broke this?

**Comment by vsoch** [comment]
@amye I don’t think anyone did anything it’s a bug in the parser code that I pointed out above. Likely the solution is to do a more scoped replacement to only the link text in question and then replace the entire block.

**Comment by vsoch** [comment]
I can futz around with it today - I wrote this whole template and the code in question so I’m probably best to take a first shot.

**Comment by nate-double-u** [comment]
Happy to help, If you need anything let me know 🙂

**Comment by vsoch** [comment]
Could definitely use a second set of eyes here! https://github.com/opencontainers/specs.opencontainers.org/pull/4

**Comment by sudo-bmitch** [comment]
Fixed by #4. Thanks @vsoch!
